azure/tf/modules/azure_hub/routetable.tf

```diff
@@ -1,3 +1,13 @@
+resource "azurerm_ip_group" "this" {
```

Comment:
At this time, this resource is used exclusively by firewall related resources. It might make sense to leave it in firewall.tf. Do you think it makes more sense in routetable.tf?

Reply:
Thanks for taking the time to review @connorbrown-db.  I'll move this back into the firewall.tf.